zerver/tests/test_user_groups.py

```diff
@@ -69,6 +69,10 @@ def assert_user_membership(
         user_ids = get_user_group_member_ids(user_group, direct_member_only=True)
         self.assertSetEqual(set(user_ids), {member.id for member in members})
 
+    def assert_member_not_in_group(self, user_group: NamedUserGroup, member: UserProfile) -> None:
+        user_ids = get_user_group_member_ids(user_group, direct_member_only=True)
+        self.assertNotIn(member.id, user_ids)
```

Comment:
Although this is used once, thought it would be good to have as a separate function for future.